app/helpers/players_helper.rb

```diff
@@ -1,2 +1,71 @@
 module PlayersHelper
+
+  def calculate_css(player)
```

Comment:
This is a little weird since player has nothing to do with css, I think it should accept a ratio between the player and the next level and be left as a page helper